projects/vdk-plugins/vdk-test-utils/src/vdk/plugin/test_utils/util_plugins.py

```diff
@@ -57,7 +57,7 @@ def new_connection(self):
         import sqlite3
 
```

Comment:
We should cleanup any hardcoded versions mentioned, for example:
* https://github.com/vmware/versatile-data-kit/blob/5b6ab858cda28f5374ec636cbf0cf8a9b63b6b5d/projects/vdk-plugins/vdk-meta-jobs/README.md?plain=1#L158 
* https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-control-cli/.gitlab-ci.yml#plain=1#L27

a link to the changing sources may be used instead

Reply:
Thanks for pointing it out. I didn't see them. I will fix those in the respective PR for the plugin